tags/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+from rest_framework_bulk.routes import BulkRouter
+
+from tags.views import NamespaceViewSet, TagViewSet, TagLinkView, get_tags_for_scope
+
+urlpatterns = [
+    path(
+        "link/<model>/<instance_pk>/tag/<tag_pk>/",
```

Comment:
You can specify the format of the capturing `<…>` (https://docs.djangoproject.com/en/2.2/topics/http/urls/#path-converters)